src/fuzzer/Types.ts

```diff
@@ -1,5 +1,13 @@
 import { ArgOptions } from "./analysis/typescript/Types";
 
+export type ResultCateogry =
```

Comment:
Think we could use this to replace the `FuzzResultCategory` enum later in this file. I'll merge the two in a subsequent PR.